### PR TITLE
[DOCS-67] SDK overview

### PR DESCRIPTION
**Pull request by harbu**

### CODE REVIEW

File: app/src/newdocs/content/SDKs.mdx [hunk 1]
```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
...
---content--
+There are both official and unofficial SDKs (Software Development Kits) available for building software that interacts
+with the Streamr platform. Using an SDK provides an effortless and reliable way to use Streamr from your software
+projects. An SDK hides the intricacies involved of interacting with Streamr behind an intuitive library API, allowing a
+developer to focus on their task at hand.
+
+
+Currently, we provide official SDKs for Javascript (browser and Node.js) and Java. There is also an experimental SDK for
+Python in the works. The range of features each of these SDKs provide may vary; be sure to check the documentation of
+a specific SDK for more information.
```
Comment (hpihkala) [inline app/src/newdocs/content/SDKs.mdx]: A  feature matrix would be nice here. Here's a starting point, feel free to improve:


|                            | [JS](https://github.com/streamr-dev/streamr-client-javascript) | [Java](https://github.com/streamr-dev/streamr-client-java) | [Python](https://github.com/streamr-dev/streamr-client-python) |
| --                         | -- | --   | --     |
| Publish                    | ✅ | ✅   | ✅     |
| Subscribe                  | ✅ | ✅   | ✅     |
| Managing Streams           | ✅ | ✅   | ✅     |
| API Key Authentication     | ✅ | ✅   | ✅     |
| Ethereum Authentication    | ✅ | ✅   | ❌     |
| Duplicate Detection        | ✅ | ✅   | ✅     |
| Gap Filling                | ✅ | ✅   | ✅     |
| Message Signing            | ✅ | ✅   | ❌     |
| Message Verification       | ✅ | ✅   | ❌     |
| End-to-end Encryption      | ✅ | ✅   | ❌     |
| Requesting Historical Data | ✅ | ✅   | ❌     |

**Comment by hpihkala** [comment]
Added @fonty1 for a big picture check and @githubwei for potential insight into external developers' point of view. For me, a compact intro works ok. Please comment if you have further ideas on what should be added.

File: app/src/newdocs/content/SDKs.mdx [hunk 2]
```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
 <ScrollableAnchor id="sdks-overview"><div>
 
 ## Overview
---content--
+There are both official and unofficial SDKs (Software Development Kits) available for building software that interacts
```
Comment (fonty1) [inline app/src/newdocs/content/SDKs.mdx]:
```suggestion
There are both official and unofficial Software Development Kits (SDKs) available for building software that interacts
```

File: app/src/newdocs/content/SDKs.mdx [hunk 3]
```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
...
+with the Streamr platform. Using an SDK provides an effortless and reliable way to use Streamr from your software
+projects. An SDK hides the intricacies involved of interacting with Streamr behind an intuitive library API, allowing a
```
Comment (fonty1) [inline app/src/newdocs/content/SDKs.mdx]:
```suggestion
projects. The SDKs take care of the small details required to interact with Streamr through intuitive API libraries, allowing a
```

File: app/src/newdocs/content/SDKs.mdx (hunk 1 above)
Comment (fonty1) [inline app/src/newdocs/content/SDKs.mdx]:
```suggestion
the specific SDK for more information.
```

File: app/src/newdocs/content/SDKs.mdx [hunk 4]
```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
...
+developer to focus on their task at hand.
+
+
+Currently, we provide official SDKs for Javascript (browser and Node.js) and Java. There is also an experimental SDK for
+Python in the works. The range of features each of these SDKs provide may vary; be sure to check the documentation of
+a specific SDK for more information.
+
+If your programming language or software environment of choice does not have an SDK available, you can still interact
+with Streamr using our [JSON-based HTTP API](api).
```
Comment (fonty1) [inline app/src/newdocs/content/SDKs.mdx]:
```suggestion
with Streamr using our <Link to={links.newdocs.api}>JSON-based HTTP API</Link> .
```

Comment (fonty1) [reply app/src/newdocs/content/SDKs.mdx]: this style of `<Link>` is better for internal redirects because it uses React Router (no page refresh).

**Comment by githubwei** [comment]
Just added content for Python SDK section

**Comment by fonty1** [comment]
All open docs PRs need to merged to keep the momentum so we'll need to follow up on those extra PR comments afterwards.

**Pull request merged**